lib/controller.py

```diff
@@ -521,3 +523,97 @@ def do_dun(self, line):
     def do_undun(self, line):
         results = self.cluster.infoUndun(self.mods['line'], nodes=self.nodes)
         self.view.dun(results, self.cluster, **self.mods)
+
+@CommandHelp('"collectinfo" is used to collect system stats on the local node.')
+class CollectinfoController(CommandController):
+    def collect_local_file(self,src,dest_dir):
+        shutil.copy2(src, dest_dir)
+        
+    def write_log(self,collectedinfo,src_file='/var/log/aerospike/*log'):
+        aslogdir = '/tmp/as_log_' + str(time.time())
+        aslogfile = aslogdir + '/ascollectinfo.log'
+        os.mkdir(aslogdir)
+        f = open(str(aslogfile), 'w')
+        f.write(str(collectedinfo))
+        f.close()
+        self.collect_local_file(src_file,aslogdir)
+        util.shell_command(["tar -czvf " + aslogdir + ".tgz " + aslogdir])
+        sys.stderr.write("\x1b[2J\x1b[H")
+        print "\n\n\nFiles in " + aslogdir + " and " + aslogdir + ".tgz saved. Please send tgz archive to support@aerospike.com"
+        print "END OF ASCOLLECTINFO"
+
+    def do_collectinfo(self, line):
+        capture_stdout = util.capture_stdout
+        collect_output = ''
+        collect_output = time.strftime("%Y-%m-%d %H:%M:%S UTC\n", time.gmtime())
+        info_params = ['network','service', 'namespace', 'xdr', 'sindex']
+        show_params = ['config', 'distribution', 'latency', 'statistics']
+        shell_cmds = ['date',
+                     'hostname',
+                     'ifconfig',
+                      'uname -a',
+                      'lsb_release -a',
+                      'ls /etc|grep release|xargs -I f cat /etc/f',
+                      'rpm -qa|grep -E "citrus|aero"',
+                      'dpkg -l|grep -E "citrus|aero"',
+                      'tail -n 10000 /var/log/aerospike/*.log',
+                      'tail -n 10000 /var/log/citrusleaf.log',
+                      'tail -n 10000 /var/log/*xdr.log',
+                      'netstat -pant|grep 3000',
+                      'top -n3 -b',
+                      'free -m',
+                      'df -h',
+                      'ls /sys/block/{sd*,xvd*}/queue/rotational |xargs -I f sh -c "echo f; cat f;"',
+                      'ls /sys/block/{sd*,xvd*}/device/model |xargs -I f sh -c "echo f; cat f;"',
+                      'lsof',
+                      'dmesg',
+                      'iostat -x 1 10',
+                      'vmstat -s',
+                      'vmstat -m',
+                      'iptables -L',
+                      'cat /etc/aerospike/aerospike.conf',
+                      'cat /etc/citrusleaf/citrusleaf.conf',
```

Comment:
Even so we should do it in a way that doesn't generate not founds. Otherwise it is confusing to the users. I've seen this a couple times with asmonitor's version.